tests/ui/test_ui.py

```diff
@@ -16,6 +16,9 @@ def mock_external_classes(self, mocker):
     @pytest.fixture
     def view(self, mocker):
         main_window = mocker.patch('zulipterminal.ui.View.main_window')
+        # View is an urwid.Frame instance, a Box widget.
+        mocker.patch.object(main_window, 'sizing',
+                            return_value=frozenset(['box']))
```

Comment:
This appears equivalent to adding a parameter to the first patch, settings sizing equal to a lambda?
Normally it seems cleaner to combine the mock setup in one line if possible.